.github/workflows/release-please.yaml

```
@@ -19,7 +19,7 @@ jobs:
     steps:
       - name: Release Please
         id: release
-        uses: googleapis/release-please-action@v4
+        uses: googleapis/release-please-action@5792afc6b46e9bb55deda9eda973a18c226bc3fc # v4.1.5
```

`googleapis/release-please-action` used version `v4.3.0` in the last workflow run:
https://github.com/aquasecurity/trivy/actions/runs/17317730860/job/49163815907#step:1:40
https://github.com/googleapis/release-please-action/releases/tag/v4.3.0


```suggestion
        uses: googleapis/release-please-action@c2a5a2bd6a758a0937f1ddb1e8950609867ed15c # v4.3.0
```